InnerEye/ML/model_training.py

```diff
@@ -53,11 +53,20 @@ def model_train(config: ModelConfigBase,
     :param config: The arguments which specify all required information.
     :param checkpoint_handler: Checkpoint handler object to find checkpoint paths for model initialization
     """
-    checkpoint_callback = ModelCheckpoint(
-        dirpath=str(config.checkpoint_folder),
-        filename='best_val_loss_checkpoint',
-        monitor=f"{VALIDATION_PREFIX}{MetricType.LOSS.value}",
-        save_last=True)
+    # Get the path to the checkpoint to recover from
+    checkpoint_path = checkpoint_handler.get_recovery_path_train()
+
+    last_checkpoint_callback = ModelCheckpoint(dirpath=str(config.checkpoint_folder),
+                                               filename='best_val_loss_checkpoint',
+                                               monitor=f"{VALIDATION_PREFIX}{MetricType.LOSS.value}",
+                                               save_last=True)
+
+    recovery_checkpoint_callback = ModelCheckpoint(dirpath=str(config.checkpoint_folder),
```

Comment:
I think we are saving ALL checkpoints here, right? I don't think we want that, this consume a huge amount of memory.